.github/workflows/github-pages.yml

```diff
@@ -37,6 +37,9 @@ jobs:
           profile: minimal
           override: true
 
+      - name: Install Foundry
```

Comment:
Alternatively I guess we could build the dev docker image and run tests through that